src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -1,71 +1,94 @@
 <%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
 <%@ page import="teammates.common.util.Const" %>
+<%@ page import="teammates.common.util.FrontEndLibrary" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
 <%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
 <%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
 
+<c:set var="cssIncludes">
+  <link rel="stylesheet" href="<%= FrontEndLibrary.HANDSONTABLE_CSS %>" type="text/css">
+</c:set>
 <c:set var="jsIncludes">
+  <script type="text/javascript" src="<%= FrontEndLibrary.HANDSONTABLE %>"></script>
   <script type="text/javascript" src="/js/instructorCourseEnrollPage.js"></script>
+  <script type="text/javascript" src="/js/instructorCourseEnrollHandsontable.js"></script>
```

Comment:
There should be only one self-authored script imported per page. Since this library is currently not used anywhere, consider moving the contents to `instructorCourseEnrollPage.js`.
In fact, you may find functions in that JS file that is no longer relevant now.